client/app/components/pages/manuscripts.component.ts

```diff
@@ -121,6 +123,11 @@ export class ManuscriptsComponent {
 					this.tService.addTask(t).subscribe(
 						r=>{							
 							alert("task created succesfuly")
+							this.tService.getTasks({}).subscribe(
```

Comment:
what is the purpose of this?

Reply:
log to see that it was created successfuly i will delete